controllers/past-grants/views/index.njk

```diff
@@ -0,0 +1,179 @@
+{% from "components/split-nav/macro.njk" import splitNav with context %}
+{% from "components/form-fields.njk" import formField with context %}
+{% from "components/promo-card/macro.njk" import promoCard %}
+{% from "components/breadcrumb-trail/macro.njk" import breadcrumbTrail %}
+
+{% extends "layouts/main.njk" %}
+{% set bodyClass = 'has-static-header' %}
+
+{% macro grantResult(grant) %}
+    <article class="grant-result">
+        <header class="grant-result__header">
+            <h3 class="grant-result__title"><a href="#">{{ grant.title }}</a></h3>
+            <p class="grant-result__summary">
+                £{{ grant.amountAwarded | numberWithCommas}} to
+                {{ grant.recipientOrganization[0].name }}
+                on {{ formatDate(grant.awardDate, DATE_FORMATS.short) }}
+            </p>
+        </header>
+    </article>
+{% endmacro %}
+
+{% block content %}
+    <main role="main">
+        <div class="content-box u-inner-wide-only accent--{{ pageAccent }} a--border-top">
+            {{ breadcrumbTrail(breadcrumbs) }}
+            <form class="grants-search" method="get" action="">
+                <h1>Search past grants</h1>
+
+                <div class="search-bar">
+                    <label class="search-bar__label" for="search-query">
+                        Search
+                    </label>
+                    <input class="search-bar__field" type="search"
+                        id="search-query" name="q"
+                        value="{{ queryParams.q }}"
+                        placeholder="Enter a search term, location, or postcode">
+                    <div class="search-bar__submit">
+                        <input class="btn btn--medium" type="submit" value="Search" />
+                    </div>
+                </div>
+
+                <div class="grants-search__totals">
+                    Showing <strong>{{ meta.totalResults | numberWithCommas }}</strong> results{% if queryParams %} {% endif %}</p>
+                </div>
+
+                <div class="grants-search__content">
+                    <div class="grants-search__results">
+                        {% if grants.length > 0 %}
+                            {% for grant in grants %}
+                                <div class="u-margin-bottom">
+                                    {# @TODO: Remove grant.plannedDates[1].endDate when import issue has been fixed  #}
```

Comment:
Think this is done now:

![image](https://user-images.githubusercontent.com/394376/46018064-f02cd280-c0d0-11e8-9e52-ea239f2e8153.png)

(I synchronised the data this morning)